### PR TITLE
Fixed typos

### PR DESCRIPTION
**Pull request by KittJonathan**

### CODE REVIEW

File: data-transform.qmd [hunk 1]
```diff
@@ -535,7 +535,7 @@ daily_flights <- daily |>
   )
 ```
 
-If you're happy with this behavior, you can explicitly request it in order to suppress the message:
+If you're not happy with this behavior, you can explicitly request it in order to suppress the message:
```
Comment (hadley) [inline data-transform.qmd:538]: I don't this should be negated; specifying `drop_last` means that you're happy with the default behaviour, you just don't want the message.

Comment (KittJonathan) [reply data-transform.qmd:538]: Makes sense :-)

Comment (mine-cetinkaya-rundel) [inline data-transform.qmd:538]:
```suggestion
If you're happy with this behavior, you can explicitly request it in order to suppress the message:
```

**Comment by mine-cetinkaya-rundel** [comment]
Thank you!

**Pull request merged**